guides/v2.0/contributor-guide/contributing_docs.md

```diff
@@ -8,26 +8,42 @@ github_link: contributor-guide/contributing_docs.md
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+The 2.0 version of Magento DevDocs has reached its end of life (EOL). The doc set will still be available and visible on the Magento DevDocs site for you to access and reference. [Magento 2.0.18](https://devdocs.magento.com/guides/v2.0/release-notes/ReleaseNotes2.0.18CE.html) is the final 2.0x release, and as of March 2018, Magento 2.0x will no longer receive security patches, quality fixes, or documentation updates.
+</div>
+
+## Contribution guideslines
+We use [Markdown](http://daringfireball.net/projects/markdown/){:target="_blank"} to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html){:target="_blank"}.
 
-## Contribution guidelines
-We use [Markdown](http://daringfireball.net/projects/markdown/){:target="\_blank"} to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html){:target="\_blank"}. We have a <a href="{{ page.baseurl }}/contributor-guide/templates/basic_template.html" target="\_blank">template</a> to make getting started easy.
+You can update or add content to existing topics. We have all topics in versioned directories for 2.0, 2.1, 2.2, and onward.
 
-{% include note.html type="info" content="Refer to the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca) for detailed information about licensing. All contributors are required to submit a click-through form to agree to the terms." %}
+We provide templates to help get you started on new topics:
+
+* [General topic template]({{ page.baseurl }}/contributor-guide/templates/basic_template.html) with example formats and styles.
+* Tutorial templates with example formats and styles for step-by-step instructions, like How Tos. It includes templates for the [first introduction topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-first.html), [middle topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-middle.html) for each step, and a [final step topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-last.html).
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+Refer to the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca) for detailed information about licensing. All contributors are required to submit a click-through form to agree to the terms.
+</div>
 
 1.  First, check [existing pull requests](https://github.com/magento/devdocs/pulls){:target=\_blank"} and make sure you are not duplicating work!
```

Comment:
Add intro sentence to this numbered list/series, and remove "first,".

"Check existing pull requests to make sure you are not duplicating work."

"This may further prevent duplicate or unnecessary effort."

Reply:
edited!